src/components/flow/routers/localization/RouterLocalizationForm.tsx

```diff
@@ -119,7 +119,9 @@ export default class RouterLocalizationForm extends React.Component<
       const { verboseName } = getOperatorConfig(originalCase.type);
 
       if (originalCase.arguments) {
-        const [orginalArgument] = originalCase.arguments;
```

Comment:
i tried googling, but couldn't find any refs or examples of declaring a variable this way `const [orginalArgument]` - is this syntax some sort of var declaration shorthand?